backend/typescript/middlewares/auth.graphql.ts

```diff
@@ -0,0 +1,99 @@
+import { AuthenticationError, ExpressContext } from "apollo-server-express";
```

Comment:
This file will be combined with backend/typescript/middlewares/auth.ts (REST version) and annotated with scrubber tags